compiler/odds.ml

```diff
@@ -8,22 +8,22 @@
  *  - Lilly Wang
  *)
 
-type action = Compile | Help | Raw
```

Comment:
I'm sure it'd be helpful but not completely convinced it's necessary. We can change the parser tests to test the SAST and the compiler tests will test the PAST. If someone wants to write it then it'd be nice but not sure it's worth our time.